daprdocs/content/en/developing-applications/building-blocks/state-management/state-management-overview.md

```diff
@@ -8,80 +8,73 @@ description: "Overview of the state management building block"
+Dapr supports optimistic concurrency control (OCC) using ETags. When a state is requested, Dapr always attaches an ETag property to the returned state. When the user code tries to update or delete a state, it’s expected to attach the ETag either through the request body for updates or the `If-Match` header for deletes. The write operation can succeed only when the provided ETag matches with the ETag in the state store.
 
-Dapr chooses OCC because in many applications, data update conflicts are rare because clients are naturally partitioned by business contexts to operate on different data. However, if your application chooses to use ETags, a request may get rejected because of mismatched ETags. It's recommended that you use a [retry policy](#retry-policies) to compensate for such conflicts when using ETags.
+Dapr chooses OCC because in many applications, data update conflicts are rare because clients are naturally partitioned by business contexts to operate on different data. However, if your application chooses to use ETags, a request may get rejected because of mismatched ETags. It's recommended that you use a retry policy to compensate for such conflicts when using ETags.
 
 If your application omits ETags in writing requests, Dapr skips ETag checks while handling the requests. This essentially enables the **last-write-wins** pattern, compared to the **first-write-wins** pattern with ETags.
 
 {{% alert title="Note on ETags" color="primary" %}}
 For stores that don't natively support ETags, it's expected that the corresponding Dapr state store implementation simulates ETags and follows the Dapr state management API specification when handling states. Because Dapr state store implementations are technically clients to the underlying data store, such simulation should be straightforward using the concurrency control mechanisms provided by the store.
 {{% /alert %}}
 
+Read the [API reference]({{< ref state_api.md >}}) to learn how to set concurrency options.
+
 ### Consistency
 
 Dapr supports both **strong consistency** and **eventual consistency**, with eventual consistency as the default behavior.
 
 When strong consistency is used, Dapr waits for all replicas (or designated quorums) to acknowledge before it acknowledges a write request. When eventual consistency is used, Dapr returns as soon as the write request is accepted by the underlying data store, even if this is a single replica.
 
-Visit the [API reference]({{< ref state_api.md >}}) to learn how to set consistency options.
+Read the [API reference]({{< ref state_api.md >}}) to learn how to set consistency options.
 
+<!---
```

Comment:
Looks like this is now hidden. I don't think we do that anywhere else in the docs. Should this section just be removed?

Reply:
Why was it hidden? I agree we shouldn't hide docs if the feature is cut

Reply:
The feature is cut. See https://github.com/dapr/docs/issues/1172.  It will resurface one day, however I agree we should remove this